main.py

```diff
@@ -164,6 +150,51 @@ def draw(self):
         img = g_font.render("Moves: " + str(self.counter), True, BLACK)
         g_screen.blit(img, (20, 20))
 
+    def move(self, ball):
+        """
+        Function Game.move(self, ball):
+        Move function, that handles handles the action when the player clicks on 
```

Comment:
Repetition corrected:
```suggestion
        Move function, that handles the action when the player clicks on 
```

Reply:
fixed